networking/src/p2p/stream.rs

```diff
@@ -116,13 +116,52 @@ impl From<TcpStream> for MessageStream {
     }
 }
 
-/// Reader of the TCP/IP connection.
-pub struct MessageReader {
+pub struct CryptData {
```

Comment:
@akoptelov 
I am not sure about this name `CryptData`, this struct is not about data, provide encrypt/decrypt function,
so maybe `Crypto` is better, it will be allign with `CryptoError` :)

Reply:
Makes sense.